Problem1.java

```diff
@@ -0,0 +1,15 @@
+// Problem1  Implement strStr() (https://leetcode.com/problems/implement-strstr/)
+
+//Time comlexity O(hlen (hlen - nlen)) 
+class Solution {
```

Comment:
its m*n brute force. For each index in haystack you are checking if the substring of needle is there or not. Try KMP